src/NHibernate.Test/NHSpecificTest/DtcFailures/DtcFailuresFixture.cs

```diff
@@ -276,6 +276,9 @@ public void CanDeleteItemInDtc()
 				}
 			}
 
+			// Dodging "latency" due to db still haven't actually committed a distributed tx after scope disposal.
+			Thread.Sleep(100);
```

Comment:
I intend to put a better "dodge" logic in #627. It will apply only to database drivers showcasing the "latency" trouble.